ml-agents/mlagents/trainers/tests/test_policy.py

```diff
@@ -18,21 +18,33 @@ def basic_params():
 def test_take_action_returns_empty_with_no_agents():
     test_seed = 3
     policy = TFPolicy(test_seed, basic_mock_brain(), basic_params())
-    no_agent_brain_info = BrainInfo([], [], [], agents=[])
-    result = policy.get_action(no_agent_brain_info)
-    assert result == ActionInfo([], [], {})
+    no_agent_step = BatchedStepResult(
```

Comment:
I think there is a method to generate an empty batchedStepResult

Reply:
Yep - using that now